src/main/java/utility/Random.java

```diff
@@ -0,0 +1,25 @@
+package utility;
+
+public class Random {
```

Comment:
Class with needed functionality already exists in apache commons
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/RandomStringUtils.html
Please use it instead of creating your own class